README.md

```diff
@@ -98,3 +98,32 @@ $di->set(
     }
 );
 ```
+
+## Slack logger
+
+Adapter to write messages into a Slack channel
+
+![slack-log](https://github.com/DirectInfoService/incubator-logger/assets/78140431/98d8515e-4fac-4904-ab63-fb114ae8b54c)
```

Comment:
Please make new local directory `.assets` and upload this image there, and use local path instead of remote.